data/coalition/coalition missions.txt

```diff
@@ -2335,3 +2335,469 @@ mission "Coalition: Submarines 4"
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris hasn't entered the system yet. Better depart and wait for it.`
+
+mission "Coalition: Expeditions Past 7"
+	landing
+	name "Curious Red Cloud"
+	description "Look for the system near Algieba where Mimodum, the expedition's leader, jumped to investigate the cosmological phenomenon in the system."
+	passengers 1
+	to offer
+		has "Coalition: Expeditions Past 6: done"
+	source "Watcher"
+	on offer
+		conversation
+			`You fly around the moon as Sedlitaris nails down the coordinates of the pylon, until you land close to one of the forests. "Deeper inside, the pylon must be. Uninhabited by humans this world is, correct?" You confirm it, and she prepares to leave. "No dragons to worry about here, there are, so try to bring this pylon back with us, I will."`
+			`	You two leave your ship and enter the forest, taking turns carrying the pylon you brought, which is easy enough to do since the gravity is so much lower here. Occasionally Sedlitaris stops and has the pylon put down so that she can determine where to go next. Eventually, you notice the trees start getting smaller as you move on, until you reach a wide, odd clearing where the ground is devoid of any of the local flora. Sedlitaris spots the other pylon on the opposite side, half stuck in the ground, and moves to pull it out. As you follow her to the machine, you notice deep, perfectly straight holes in the ground, forming lines that sometimes cross the entire clearing and continue well past it. Soil and dust have filled most of them up, but in certain gaps you can look down to cracks that must be dozens of meters deep.`
+			`	Sedlitaris ties a rope around the end of the pylon that's visible and pulls it, forcing up the dirt as she brings the rest of the device to surface. She connects it with the pylon you brought, and accesses the data from there. "Much more active here, the expedition was," she says. "Three recordings, there are. Bring these back to your ship, we should, listen to the first one there, we can."`
+			`	Both of you carrying a pylon, you make your way out of the forest and back to your ship, where Sedlitaris immediately plays the oldest entry: "Log started. Send two Punishers further north, I did, but returned yet, they have not. As for the crimson anomaly on the nearby western system, still unresponsive, it is. A reaction from it, nothing got, be it a Finisher barrage or attempting to ram it. A simple nebula, the researchers claim it is not, think that we are missing something, they do. Wait a few more days for the Punishers to return, I will. In the meantime, back to that system, I will go. Maybe learn something about the anomaly, we can."`
+			`	After Mimodum's voice has disappeared, Sedlitaris urges you to take off. "Heard of this 'anomaly' before, I had not. Look for the system it's in, at once we must. Hopefully still there after the centuries, it is."`
```

Comment:
It was meant to be more a gameplay thing yes. Either you don't know where Terminus is, and have to pay attention to what the recording says to find it, or you'll know what they're talking about from the description of the anomaly.
Also, potentially something to serve as a pointer to Terminus in case one hasn't found it yet.